app/components/AddToCalender/AddToCalender.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from 'packages/lego-bricks/src/components/Button';
+
+type AddToCalenderProps = {
+  title: string;
+  startTime: string; // Format: YYYYMMDDTHHMMSSZ
+  endTime: string; // Format: YYYYMMDDTHHMMSSZ
```

Comment:
```suggestion
  startTime: Dateish;
  endTime: Dateish;
```
`import type { Dateish } from 'app/models';`